elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpers.java

```diff
@@ -0,0 +1,124 @@
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@Slf4j
+/**
+ * Util class for Dynamic config helper module.
+ */
+public class DynamicConfigHelpers {
+
+    private static final String TABLE_CONFIG_PATH = "tables/";
+    private static final String SECURITY_CONFIG_PATH = "security.hjson";
+    private static final String VARIABLE_CONFIG_PATH = "variables.hjson";
+    private static final String NEW_LINE = "\n";
+
+
+    /**
+     * Checks whether input is null or empty.
+     * @param input : input string
+     * @return true or false
+     */
+    public static boolean isNullOrEmpty(String input) {
+        return (input == null || input.trim().length() == 0);
+    }
+
+    /**
+     * format config file path.
+     * @param basePath : path to hjson config.
+     * @return formatted file path.
+     */
+    public static String formatFilePath(String basePath) {
+        if (!basePath.endsWith("/")) {
+            basePath += '/';
+        }
+        return basePath;
+    }
+
+    /**
+     * converts variable.hjson to map of variables.
+     * @param basePath : root path to model dir
+     * @return Map of variables
+     * @throws JsonProcessingException
+     */
+    public static Map<String, Object> getVaribalesPojo(String basePath) throws JsonProcessingException {
+        String jsonConfig = hjsonToJson(readConfigFile(new File(basePath + VARIABLE_CONFIG_PATH)));
+        return getModelPojo(jsonConfig, Map.class);
+    }
+
+    /**
+     * converts all avaiable table config to ElideTableConfig Pojo.
+     * @param basePath : root path to model dir
+     * @return ElideTableConfig pojo
+     * @throws JsonProcessingException
+     */
+    public static ElideTableConfig getElideTablePojo(String basePath) throws JsonProcessingException {
+        Collection<File> tableConfigs = FileUtils.listFiles(new File(basePath + TABLE_CONFIG_PATH),
+                new String[] {"hjson", "json"}, false);
+        Set<Table> tables = new HashSet<>();
+        for (File tableConfig : tableConfigs) {
+            String jsonConfig = hjsonToJson(readConfigFile(tableConfig));
+            ElideTableConfig table = getModelPojo(jsonConfig, ElideTableConfig.class);
+            tables.addAll(table.getTables());
+        }
+        ElideTableConfig elideTableConfig = new ElideTableConfig();
+        elideTableConfig.setTables(tables);
+        return elideTableConfig;
+    }
+
+    /**
+     * converts security.hjson to ElideSecurityConfig Pojo.
+     * @param basePath : root path to model dir.
+     * @return ElideSecurityConfig Pojo
+     * @throws JsonProcessingException
+     */
+    public static ElideSecurityConfig getElideSecurityPojo(String basePath) throws JsonProcessingException {
+        String jsonConfig = hjsonToJson(readConfigFile(new File(basePath + SECURITY_CONFIG_PATH)));
```

Comment:
we can do a check here instead if the file exists or does not exists and log message with results. Then make a call to readConfig method. Right now readConfigFile will return with a exception if file does not exist and print an error message which could be misleading. Same is applicable for Variables.